assets/css/global.css

```diff
@@ -0,0 +1,14 @@
+#map {
```

Comment:
I think we have to think about how to keep the styles and the js components together, so we don't have to add the css separately on each feature, in the meantime, let's move this styles to the feature css file view-hazard/style.css